cpu/avr8_common/Makefile

```diff
@@ -0,0 +1,11 @@
+# define the module that is build (not strictly necessary)
+MODULE = avr8_common
+
+# add a list of subdirectories, that should also be build
+DIRS = avr_libc_extra
+
+ifneq (,$(filter cpu_avr8_common_cxx,$(USEMODULE)))
+  DIRS += cxx
+endif
```

Comment:
This is either a leftover from the past or it slipped in by accident. In either case this can be dropped now. See https://github.com/RIOT-OS/RIOT/pull/15713

Reply:
Could you rebase on top of current master, as now #15713 is upstream?

Reply:
Nice, done!

Since I'm new on RIOT, I noted that static tests are failing because of fixups and I'm not sure if I'm doing everything right.
If I missed something with fixups, just let me know.

Reply:
> Since I'm new on RIOT, I noted that static tests are failing because of fixups and I'm not sure if I'm doing everything right.

Everything is right. This is just to prevent maintainers from accidentally hitting the merge button before you had the chance to squash the fixups as intended.